src/vos/tests/evt_ctl.c

```diff
@@ -387,13 +411,15 @@ ts_find_rect(char *args)
 	struct evt_entry_array	 ent_array;
 	int			 rc;
 	bool			 should_pass;
+        char                    *args;
 
+        args = tst_fn_val.optval;   
```

Comment:
(style)  trailing whitespace
(style)  code indent should use tabs where possible